### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D: Control type provided for Target Database in incorrect

### ISSUE THREAD

**Issue by AbhiramDeshmukh** [microsoft/azuredatastudio#6691]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags: #A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #AI4D; #MAS4.1.2; MAS4.2.1;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name: Accessibility Insights tool
MAS References: MAS4.1.2, MAS1.3.1, MAS4.2.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Expand the server connected in step 2.
5. Right click an on prem database and select Restore from Context menu.
6. Start Accessibility Insights for windows tool.
7. Check the control type provided for Target Database.

**Actual:**
Control type is given as "Edit" for Target Database.

**Expected:**
For Target Database, control type should be combo box.

**User Impact:**
As control type is given as Edit & then there is separate control type for Down Arrow, keyboard only users require one extra tab to navigate to next step. Also If we use Alt+DownArrow when focus shifts to this control (Edit), options are shown. Similarly, if we hit Enter when focus shifts to down arrow present next to this Edit control same options are shown. If users are getting same data from two controls then it will confuse users.
The Down arrow shown is not having name associated to it which will impact screen reader users.

**Recommendation:**
Please refer to below URL which is repository for accessibility bug fixes.
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS 4.1.2: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
MAS 4.2.1: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}
MAS 1.3.1: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for reference:** 
![6691_A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D](https://user-images.githubusercontent.com/31536050/62858783-e5932000-bd18-11e9-93be-3df5fe5dcec6.png)

**Comment by udeeshagautam** [comment]
As per our discussion it should be okay to say edit for editable control. Will discuss in Thursday triage.

**Comment by kenvanhyning** [comment]
Feature not moving to GA in current planning. Closing as won't fix.
